src/main/java/bitmap/transformer/App.java

```diff
@@ -16,14 +16,14 @@ public String getGreeting() {
     }
 
     public static void main(String[] args) {
-        readBitMap();
         System.out.println(new App().getGreeting());
+        readBitMap();
 
     }
         public static void readBitMap() {
 
-           BitMap test = new BitMap("src/main/resources/mario.bmp", "src/main/resources/output.bmp");
-            test.negativeRGB();
+           Bitmap test = new Bitmap("src/main/resources/mario.bmp", "src/main/resources/output.bmp");
```

Comment:
To refactor this to run using command line arguments, use the `args` String array. So this line would be `Bitmap test = new Bitmap(args[0], args[1], args[2]);`

Reply:
`args[2]` will be the third argument received from the command line, we want that to be the transform the user wants to do. You can add some logic in order to figure out what method to call. Like `if(args[2].equals("negative") { test.negativeRGB(); }`